package.json

```diff
@@ -4,7 +4,7 @@
  "displayName": "Upbound",
  "preview": true,
  "publisher": "upboundio",
- "version": "0.0.4",
+ "version": "0.0.5",
```

Comment:
Do we need to bump version for this? Also, I was looking at the link to `CONTRIBUTING.md` in https://marketplace.visualstudio.com/items?itemName=Upboundio.upbound and I think the link format causes it to lead to the wrong place.

Reply:
>Do we need to bump version for this?

As we discussed offline, I was going to say "no". However, the fix for the link in the description in the vscode marketplace will need us to publish a new version.